sdk/cosmosdb/ci.yml

```diff
@@ -0,0 +1,40 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+      - master
+      - feature/*
+  paths:
+    include:
+      - sdk/cosmosdb/
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+  paths:
+    include:
+      - sdk/cosmosdb/
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+    parameters:
+      ServiceDirectory: cosmosdb
+
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
+    parameters:
+      PackagePath: "sdk/cosmosdb/cosmos/"
+      Toolchain: npm
+      Matrix:
+        Windows_Node8:
+          OSVmImage: "vs2017-win2016"
+          NodeVersion: "8.x"
```

Comment:
The previous ci.yml used Node 6. Does the Cosmos team no longer want to validate against Node 6?

Reply:
The public pipelines in azure-cosmos-js repo run on Node 8 and not Node 6.

Reply:
I can double-check with them.

Reply:
8 is fine. We only needed 6 for v2